src/elfpy/agent.py

```diff
@@ -64,8 +74,22 @@ def get_max_pt_short(self) -> float:
         """
         if self.market.share_reserves == 0:
             return 0
-        max_pt_short = self.market.share_reserves * self.market.share_price / self.market.get_spot_price()
-        return max_pt_short
+        time_remaining = time_utils.get_yearfrac_remaining(
+            self.market.time, self.market.time, self.market.token_duration
```

Comment:
can you name these args, so that they're more clear. Since the values are the same.